go.mod

```diff
@@ -3,17 +3,16 @@ module github.com/palomachain/paloma
 go 1.19
 
 require (
-	github.com/CosmWasm/wasmd v0.30.0
+	github.com/CosmWasm/wasmd v0.11.1
 	github.com/CosmWasm/wasmvm v1.1.1
-	github.com/cosmos/cosmos-sdk v0.45.11
-	github.com/cosmos/ibc-go/v4 v4.2.0
+	github.com/cosmos/cosmos-sdk v0.39.3
```

Comment:
Same as https://github.com/palomachain/paloma/pull/742 - CosmosSDK version should not have been bumped as it is backward incompatible with current one